setup.py

```diff
@@ -0,0 +1,53 @@
+from setuptools import setup
+from codecs import open
+from os import path
+
+here = path.abspath(path.dirname(__file__))
+
+# Get the long description from the relevant file
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+setup(
+    name='django-see-profile',
+    # Versions should comply with PEP440.  For a discussion on single-sourcing
+    # the version across setup.py and the project code, see
+    # http://packaging.python.org/en/latest/tutorial.html#version
+    version='1.0.1',
+    description='Scoped thread-local mixin class',
+    long_description=long_description,
+    # The project's main homepage.
+    url='https://github.com/sdelements/django-see-profile',
+    # Author details
+    author='Security Compass',
+    author_email='jeff@securitycompass.com',
+    # Choose your license
+    license='MIT',
+    # See https://pypi.python.org/pypi?%3Aaction=list_classifiers
+    classifiers=[
+        # How mature is this project? Common values are
+        #   3 - Alpha
+        #   4 - Beta
+        #   5 - Production/Stable
+        'Development Status :: 4 - Beta',
+        # Indicate who your project is intended for
+        'Intended Audience :: Developers',
+        'Topic :: Software Development :: Libraries :: Python Modules',
+        # Pick your license as you wish (should match "license" above)
+        'License :: OSI Approved :: MIT License',
+        # Specify the Python versions you support here. In particular, ensure
+        # that you indicate whether you support Python 2, Python 3 or both.
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
+    ],
+    # What does your project relate to?
+    keywords='django middleware profiling',
+
+    # You can just specify the packages manually here if your project is
+    # simple. Or you can use find_packages().
+    packages=['see_profile'],
+    install_requires=[
+        "django>=1.10"
+        "django-security>=0.10.0"
```

Comment:
No it doesn't?

Reply:
I wasn't sure exactly what version of django security this was relying on. To be honest it only relies on the BaseMiddleware provided by DjangoSecurity, as that provides a bunch of useful settings validation & fetching.

In an ideal world we'd probably pull that out into a separate useful thing & both django-security & this library would depend on it. 

I could check to see what version of django security we introduced BaseMiddleware in though.

Reply:
Oh I was being funny because I thought this was a mistake. I didn't realize we had stuff you needed in django-security. Yeah should decouple things more, but that can be a thing we do next.

Reply:
I could probably drop this to >=0.9, as we haven't touched the BaseMiddleware in 5 years or so and 0.9 is about 4 years old, so it's a decent "have at least this" point.

If we really don't like the dependency it might also be worth not being so finicky about the settings info, as I can just default the behaviour to "turn it off if we're missing settings". The current form does inform people they're missing the relevant setting, but adding the dependency was a lot more obvious when this was just purely internal to the product and we knew django-security would always be there.